### PR TITLE
chore(docs): Release Notes for 4.6

### PR DESCRIPTION
**Pull request by marvinjude**
## Description

[View Release Notes](https://github.com/gatsbyjs/gatsby/blob/release-notes-4.6/docs/docs/reference/release-notes/v4.6/index.md)

### CODE REVIEW

File: docs/docs/reference/release-notes/v4.6/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+date: "2022-01-25"
+version: "4.6.0"
+title: "v4.6 Release Notes"
+---
+
+Welcome to `gatsby@4.6.0` release (January 2022 #2)
+
+Key highlights of this release:
+
+- [Performance update: Reuse cache across instances](#reuse-cache-across-instances).
```
Comment (LekoArts) [inline docs/docs/reference/release-notes/v4.6/index.md:11]:
```suggestion
- [Speeding Up Subsequent Queries](#speeding-up-subsequent-queries)
```

We'll need to tell people what benefit they get without technical terms

File: docs/docs/reference/release-notes/v4.6/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- [Improve live editing experience for images in markdown files](#improve-live-editing-experience-for-image-in-markdown-files)
+
+## Reuse cache across instances
```
Comment (LekoArts) [inline docs/docs/reference/release-notes/v4.6/index.md:14]:
```suggestion
## Speeding Up Subsequent Queries
```

We want to use https://en.wikipedia.org/wiki/Title_case with the Chicago Manual of Style

File: docs/docs/reference/release-notes/v4.6/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+version: "4.6.0"
+title: "v4.6 Release Notes"
+---
+
+Welcome to `gatsby@4.6.0` release (January 2022 #2)
+
+Key highlights of this release:
+
+- [Performance update: Reuse cache across instances](#reuse-cache-across-instances).
+- [Improve live editing experience for images in markdown files](#improve-live-editing-experience-for-image-in-markdown-files)
```
Comment (LekoArts) [inline docs/docs/reference/release-notes/v4.6/index.md:12]:
```suggestion
- [Tracking Image Changes in Markdown Files](#tracking-image-changes-in-markdown-files)
```

File: docs/docs/reference/release-notes/v4.6/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+[Previous release notes](/docs/reference/release-notes/v4.5)
+
+[Full changelog][full-changelog]
+
+---
+
+## Notable Bugfixes & Improvements
+
+- `gatsby-transform-remark` and `gatsby-remark-images`: Fixed caching issue where `gatsby-transformer-remark` would not reload image dependencies from `gatsby-remark-images` while running `gatsby develop`, via [PR #34433](https://github.com/gatsbyjs/gatsby/pull/34433)
```
Comment (LekoArts) [inline docs/docs/reference/release-notes/v4.6/index.md:35]: This doesn't need to be here as it'll already be in the main section

File: docs/docs/reference/release-notes/v4.6/index.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+- `gatsby-plugin-utils` fixes to plugin schema validation via [PR #34182](https://github.com/gatsbyjs/gatsby/pull/34182):
+  - `pluginOptionsSchema` will give warnings instead of throw errors when a plugin receives an uknown configuration key
+  - `pluginOptionsSchema` will respect default configuration keys set with Joi
+  - `testPluginOptionsSchema` function will also return `hasWarnings` and `warnings` properties for use in test suites
```
Comment (LekoArts) [inline docs/docs/reference/release-notes/v4.6/index.md:39]: This is a breaking change, so it should be its own heading and a key highlight of the release

File: docs/docs/reference/release-notes/v4.6/index.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+Welcome to `gatsby@4.6.0` release (January 2022 #2)
+
+Key highlights of this release:
+
+- [Performance update: Reuse cache across instances](#reuse-cache-across-instances).
+- [Improve live editing experience for images in markdown files](#improve-live-editing-experience-for-image-in-markdown-files)
+
+## Reuse cache across instances
+
+Walking nodes to track inline objects is very expensive. Caching this across instances of the graphql query sped up subsequent queries by ~10-15% (depending on complexity of nodes). Every performance improvement is important to use as we constantly look for way to make Gatsby faster than ever 🚀
```
Comment (LekoArts) [inline docs/docs/reference/release-notes/v4.6/index.md:16]: The first two sentences will most people probably not understand as they don't know what nodes, inline objects etc. are. The last sentence can be deleted, we don't have to type this out.

Generally speaking you don't just want to copy the description from the PR but frame it in another way so that all users can directly get it.

That's why we have the internal doc with this quote:

```
Good questions for Release Notes:

- Who is the target audience?
- What changed in this release? Why?
- How does the change impact the user?
- Is the tl;dr clear?

tl;dr: "Loads faster with fewer crashes"
Detail: "Added list virtualization. Reduced OOMs by 20%" etc.
```

If you're not sure how to exactly write it, you'll have to reach out to the author of the PR (in this case @KyleAMathews) to help you write it.

**Review by LekoArts — CHANGES_REQUESTED** [review]

**Comment by LekoArts** [comment]
You've also committed two unrelated files you'll need to remove

**Comment by marvinjude** [comment]
@LekoArts Thanks for the comments! 

Does this look good now? Are there other parts we can Improve on? Not sure If I completely captured @j0sh77 work in #34433

**Review by j0sh77 — APPROVED** [review]
Looks good to me! 🥳
